Makefile

```
@@ -53,6 +53,10 @@ KUBEBUILDER := $(TOOLS_BIN_DIR)/kubebuilder
 KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
 ENVSUBST_BIN := envsubst
 ENVSUBST := $(TOOLS_BIN_DIR)/$(ENVSUBST_BIN)-drone
+SETUP_ENVTEST = $(TOOLS_BIN_DIR)/setup-envtest
+
+# Used for envtest
+KUBERNETES_VERSION := 1.23.x
```

This might override the env var  `KUBERNETES_VERSION` exported in other places ? 

```suggestion
ENVTEST_K8S_VERSION := 1.23.x
```